.sequelizerc

```diff
@@ -0,0 +1,8 @@
+const path = require('path')
```

Comment:
We should have this using ES6 module syntax. After all, we are using TS to be able to use the latest features of JS, right?

Reply:
@yegow I agree